diff --git a/wscript b/wscript
index 95f886737..2495bd930 100644
--- a/wscript
+++ b/wscript
@@ -1018,7 +1018,7 @@ def configure(cfg):
     if (cfg.env['DEST_OS'] == LINUX_OS or cfg.env['DEST_OS'] == FREEBSD_OS):
         cfg.check_cxx(lib='dl', cflags='-Wall', uselib_store='DL')
         cfg.check_cxx(lib='elf', cflags='-Wall', uselib_store='ELF')
-    cfg.check_cxx(lib='ncurses', cflags='-Wall', uselib_store='NCURSES')
+#    cfg.check_cxx(lib='ncurses', cflags='-Wall', uselib_store='NCURSES')
 #    cfg.check_cxx(stlib='lldb', cflags='-Wall', uselib_store='LLDB')
     cfg.check_cxx(lib='m', cflags='-Wall', uselib_store='M')
     if (cfg.env['REQUIRE_LIBFFI'] == []):
